src/vanilla/Model/CodeModelTS.cs

```diff
@@ -1167,7 +1166,7 @@ public string GenerateRollupConfig()
                 });
                 config.ArrayProperty("plugins", plugins =>
                 {
-                    plugins.Text("nodeResolve({ module: true })");
```

Comment:
mainfields should include `main`. Probably want ['module', 'main'] (and ['browser', 'module', 'main'] for the browser config).

Reply:
Thanks for catching that!  Since we only use `module` and `main` in the generated `package.json`, I've set this field to contain those two entry points and regenerated the test packages to reflect it.